infinitory/inventory.py

```diff
@@ -35,36 +40,51 @@ def query_resources(self, pdb, condition, include_absent=False):
             except KeyError:
                 continue
 
-    def load_backups(self, pdb):
-        for node, resource in self.query_resources(pdb, 'type="Backup::Job"'):
+    def load_backups(self, pupdb):
+        for node, resource in self.query_resources(pupdb, 'type="Backup::Job"'):
             paths = resource["parameters"]["files"]
             if type(paths) is list:
                 node["other"]["backups"].extend(paths)
             else:
                 node["other"]["backups"].append(paths)
 
-    def load_logging(self, pdb):
-        for node, resource in self.query_classes(pdb, "Profile::Logging::Rsyslog::Client"):
+    def load_errors(self, pupdb):
+        print("Loading errors")
```

Comment:
Please use logging, and have this off by default. By default infinitory should only produce output for warnings and errors (it's intended to be used from a task runner).

Reply:
Just going to delete it.